proto/elys/stablestake/pool.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package elys.stablestake;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/elys-network/elys/x/stablestake/types";
+
+message Pool {
```

Comment:
I think we should change this name, borrowedPool or something else, Pool is not good as when we extend this functionality it will be confusing

Reply:
Yeah, we use this structure for generalized lp pool as well, can add extra members for it in the same

Reply:
generalized lp will have one asset per pool

Reply:
so we can;t use this, we are going to deposit a single asset for a single pool id, as of now current logic is based on two assets per pool, also liabilities pool and deposit pool are two different things